### ISSUE TITLE
License ambiguity prevents efficient reuse

### ISSUE THREAD

**Issue by mtp401** [sloboste/dotfiles#1]
I'd really like to use parts of this great code base, but, I'm wondering about the legal ramifications of doing so due to the lack of a license. I would highly recommend adding a well know license such as MIT, Apache 2.0, or WTFPL. Thanks!

**Comment by sloboste** [comment]
Well first of all, the code in this repo is a total :dumpsterfire: (notably `outletctl` and `dokermit.kermit`, like seriously what kind of amateur wrote that garbage?) and I pretty much just copypasta'd most of it from the interwebs anyway. Secondly, everyone (well except for total n00bs like you) knows that no one really writes their own code anymore - everything nowadays is just shit from stackoverflow glued together with a few lines of JS (best language ever invented btw), so there's no point in licensing code.

TLDR that's never going to happen and if you were a real hacker you would have already known that. 

[Mic drop](http://gifimage.net/wp-content/uploads/2017/02/Mic-Drop-GIF-Image-7-1.gif)

**Comment by ssloboda** [comment]
@mtp04 do you even program bro? Lol

**Comment by sloboste** [comment]
:sadparrot:
